_data/2020/locations.yml

```diff
@@ -0,0 +1,168 @@
+- year: 2020
```

Comment:
Why does this live in the `2020` directory but include all year locations? Here are three other options that I think are better (with their own trade-offs):
1. One global `locations.yml` file in `_data/`.
2. A separate `locations.yml` file for each year that only contains location data for that year. (Less likely that folks will mess up prior years).
3. Each location has their own `location.yml` with this data. This is slightly harder to iterate through (you could do it with [this technique](https://github.com/compsocialscience/summer-institute/blob/4438440d548c65e2af6852d113d7c94b6f234112/_layouts/all_time_participants.html#L13-L22), but makes it more likely that any changes will only mess up one location.

Reply:
Went with recommendation `#3`. Was a bit trickier to implement than I initially thought, but the new locations page and it's corresponding sidebar has been implemented in: https://github.com/compsocialscience/summer-institute/pull/1240/commits/2b0e05f08bb62377a87b86e78e6fad449f4ca8e8 and https://github.com/compsocialscience/summer-institute/pull/1240/commits/a461342bc65c191e50644eafa1d391c427c52568